packages/wasi/lib/wasi.ts

```diff
@@ -421,6 +422,7 @@ class WASI {
 
     const CHECK_FD = (fd: number, rights: BigIntPolyfillType) => {
       const stats = stat(this, fd);
+      // console.log(`CHECK_FD: stats.real: ${stats.real}, stats.path:`, stats.path);
```

Comment:
Let's remove commented console logs 😄

Reply:
This is incredibly useful for me each time I debug.

I removed it before, but each time I try to fix something I end up rewriting it again and again.
Do you feel ok with keeping it for now and removing once WASI is fully working?

Reply:
@syrusakbary Sure! 😄